modules/ROOT/pages/log-points.adoc

```diff
@@ -1,17 +1,137 @@
 = Anypoint Monitoring Log Points
 
-The Anypoint Monitoring log points feature enables you to interactively extract data from running managed APIs. You can specify properties (for example, the host and method in the request header) in your managed API for which to generate logs without having to write code or redeploy your application. This helps you troubleshoot without having to restart the application. 
+== Log Points for Apps
+
+The Log Points feature for apps enables you to log information from connectors in any Mule app in real time. If you are using one of the supported connectors, you can select a log level (debug, info, warn, or error) and generate logs.
+
+=== Prerequisites
+
+* A https://www.mulesoft.com/anypoint-pricing[Titanium subscription] to Anypoint Monitoring and Mule 3.8.5 or later
+* The *Runtime Manager > Manage Settings* permission
+
+=== Supported Connectors
+
+The following Anypoint Connectors are available for use with the Log Points feature.
+
+.Supported Connectors
+|===
+|Connector |Supported for Mule 3.x |Supported for Mule 4.x
```

Comment:
Another way to avoid repeating is to include Anypoint Connector in the column title: "Anypoint Connector for..."
and then have entries like "Amazon S3 (Amazon S3 Connector)".
Would that work for things like Object Store: "Object Store (Object Store Connector)"?
Then we could be more parallel.